File: Java Sept 2023/OOP-II/point.java
class Point{
	private int x;
	private int y;
	public Point(){}
	public Point(int a,int b){
		x = a;
		y = b;
	}
	public Point(Point p){
		this.x = p.x;
		this.y = p.y;
	}
	
	public void display(){
		System.out.println("["+x+","+y+"]");
	}
	
	
}

File: CPP Sept 2023/oop - 1/point.cpp
#include<iostream>
using namespace std;
class Point{
    int x;
    int y;
    public:
    void accept();
    void display();
};
void Point:: accept(){
        int a,b;
        cout<<"Enter x and y cordinates: ";
        cin>>a>>b;
        x = a;
        y = b;

    }
void Point:: display(){
        cout<<"["<<x<<","<<y<<"]"<<endl;
    }
int main()
{
    Point p1;
    p1.accept();
    p1.display();


}